jabgui/src/main/java/org/jabref/gui/frame/JabRefFrame.java

```diff
@@ -643,6 +643,11 @@ public Stage getMainStage() {
 
     @Override
     public void handleUiCommands(List<UiCommand> uiCommands) {
+        if (uiCommands.stream().anyMatch(UiCommand.Focus.class::isInstance)) {
+            mainStage.toFront();
+            mainStage.requestFocus();
```

Comment:
@InAnYan I think, it is either or here.

VS.Code just lets blink the app in the task bar (which is `requestFocus` IMHO). Here, the current instance is brought to front. I assume that `requestFocus` is made in addition to ensure it works?

Reply:
I removed the `requestFocus` statement - and it worked, too.